tasks/sync/sync.thor

```diff
@@ -3,83 +3,66 @@ require 'docker-sync/config'
 require 'docker-sync/preconditions'
 require 'docker-sync/update_check'
 require 'docker-sync/upgrade_check'
+require 'daemons'
 
 class Sync < Thor
 
   class_option :config, :aliases => '-c',:default => nil, :type => :string, :desc => 'Path of the docker_sync config'
   class_option :sync_name, :aliases => '-n',:type => :string, :desc => 'If given, only this sync configuration will be references/started/synced'
 
   desc 'start', 'Start all sync configurations in this project'
+  method_option :daemon, :aliases => '-d', :default => false, :type => :boolean, :desc => 'Run in the background'
+  method_option :app_name, :aliases => '--name', :default => 'dsync', :type => :string, :desc => 'App name used in PID and OUTPUT file name for Daemon'
```

Comment:
Apologies in advance if this isn't a reasonable request - I'm not as familiar with daemonizing ...

One thing I've noticed while using this branch in comparison to using master is that you _have_ to specify names if you're working on more than one project at the same time (such as a micro-service stack). In the original approach it wouldn't matter, but with this approach you cannot simply replace `docker-sync start` with `docker-sync start -d` if you plan on utilizing more than one project.

Is it possible to utilize the name of the first defined configuration as the default name for the PID, rather than resorting to just `dsync`?